content/en/network_monitoring/devices/ping.md

```diff
@@ -65,7 +65,7 @@ instances:
 
 {{% tab "Autodiscovery" %}}
 
-To apply ping settings to all _Autodiscovery_ subnets, create the ping configuration under the `network_devices.autodiscovery` section.
+Configure ping settings in the `network_devices` section under the autodiscovery parameter to apply them to _all_ Autodiscovery subnets.
```

Comment:
Does the first mention of autodiscovery need to be capitalized?

Reply:
well wait now that I read the vale rule, they both can be lowercased , let me fix this!